CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ FetchContent_Declare(ios-cmake
     GIT_TAG 4.5.0)
 FetchContent_Declare(JsRuntimeHost
     GIT_REPOSITORY https://github.com/BabylonJS/JsRuntimeHost.git
-    GIT_TAG 27f17e531c6521b9210cfa651b43251a7c860ac6)
+    GIT_TAG 8c8e577d72524ed88b07872da612193700def1e1)
```

Comment:
This probably needs to be updated again after the JsRuntimeHost PR merges.

Reply:
fixed